Modules/Core/Services/Migration/Migrators/ClientMigrator.php

```
@@ -0,0 +1,270 @@
+                    $context->mapId('clients', $v1Id, (int) $v1Id);
+                }
+                $migrated++;
+                continue;
+            }
+
+            try {
+                $displayName = $name !== '' ? $name : $surname;
+                $active      = (bool) ($row['client_active'] ?? true);
+                $vatId       = ! empty($row['client_vat_id']) ? (string) $row['client_vat_id'] : null;
+                $taxCode     = ! empty($row['client_tax_code']) ? (string) $row['client_tax_code'] : null;
+                $website     = ! empty($row['client_web']) ? (string) $row['client_web'] : null;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$website'. (undefined)

(UnusedLocalVariable)